plugin/trino-vertica/src/test/java/io/trino/plugin/vertica/TestVerticaTypeMapping.java

```diff
@@ -149,6 +153,78 @@ public void testDouble()
                 .execute(getQueryRunner(), trinoCreateAsSelect("trino_test_double"));
     }
 
+    @Test
+    public void testTimestampFromVertica()
+    {
+        testTimestampFromVertica(UTC);
+        testTimestampFromVertica(ZoneId.systemDefault());
+        // no DST in 1970, but has DST in later years (e.g. 2018)
+        testTimestampFromVertica(ZoneId.of("Europe/Vilnius"));
+        // minutes offset change since 1970-01-01, no DST
+        testTimestampFromVertica(ZoneId.of("Asia/Kathmandu"));
+        testTimestampFromVertica(TestingSession.DEFAULT_TIME_ZONE_KEY.getZoneId());
+    }
+
+    public void testTimestampFromVertica(ZoneId sessionZone)
+    {
+        Session session = Session.builder(getSession())
+                .setTimeZoneKey(TimeZoneKey.getTimeZoneKey(sessionZone.getId()))
+                .build();
+
+        SqlDataTypeTest.create()
+                // before epoch
```

Comment:
Can we add the following roundtrips? 
* min and max timestamp values in Vertica https://docs.vertica.com/23.3.x/en/sql-reference/data-types/datetime-data-types/timestamptimestamptz/#limits
* julian->gregorian switch 1582 Oct 5-14